tests/test_broker.py

```diff
@@ -96,3 +96,34 @@ async def test_list_queue_broker(
     worker1_task.cancel()
     worker2_task.cancel()
     await broker.shutdown()
+
+
+@pytest.mark.anyio
+async def test_list_queue_cluster_broker(
+    valid_broker_message: BrokerMessage,
+    redis_cluster_url: str,
+) -> None:
+    """
+    Test that messages are published and read correctly by ListQueueClusterBroker.
+
+    We create two workers that listen and send a message to them.
+    Expect only one worker to receive the same message we sent.
+    """
+
+    print(f"redis_cluster_url: {redis_cluster_url}")
+    broker = ListQueueClusterBroker(
+        url=redis_cluster_url, queue_name=uuid.uuid4().hex
+    )
+    worker1_task = asyncio.create_task(get_message(broker))
```

Comment:
Why have you created 2 tasks here? I guess that one task is enough to test if it works.

Reply:
Agree.  I've updated the test to only use one task